content/docs/welcome-guide/create/creating-projects-using-project-manager.md

```diff
@@ -12,15 +12,19 @@ This tutorial provides an introduction to project configuration and building in
 * Create a new O3DE project.
 * Build the O3DE project.
 
-At the end of the tutorial you'll have a new O3DE project based on the default project template, ready to open in the O3DE **Editor**.
+At the end of the tutorial you'll have a new O3DE project, based on the default "standard" project template, ready to open in **O3DE Editor**.
```

Comment:
```suggestion
At the end of the tutorial you'll have a new O3DE project, based on the default "standard" project template, ready to open in the **O3DE Editor**.
```

Also same note on the "standard" wording as above.

Reply:
When formally referring to O3DE tools like this, the formatting style, similar to Lumberyard, is to leave off the preceding article, "the".

Reply:
@paucom , your opinion?